src/index.scss

```diff
@@ -11,3 +11,16 @@ body {
   max-width: 500px;
   margin: 0 auto;
 }
+
+
+.circle5 {
```

Comment:
What does circle5 mean? This naming can be improved

Reply:
fixed :(, renamed to .circle